### DISCUSSION TITLE
facing issue while submitting explorer details

### DISCUSSION THREAD

**Discussion by SardarShahzeb** [WalletConnect/walletconnect-monorepo#3785] [Developer Support]
i have created a project and i have submitted all details it got rejected and give following error
Changes requested

Your update has been rejected. Reason:

"Dapp Testing - Could not verify if testing has been done on this dapp. As we do not have available V2 metric data for Sign. Please test your integration with our test wallet and submit for approval if successful. Link: https://react-web3wallet.vercel.app"
now i have gone through above link and entered the uri and click on approve and wallet is connect with my daap .
can you please guide me what should i do to publish this so it will be accepted

**Comment by glitch-txs** [comment]
hey if you've done the test should pass the approval.

**Reply by SardarShahzeb** [reply]
yes its working but i have a question why its not approving my on explorer

**Reply by glitch-txs** [reply]
can you share your project ID or email account?

**Reply by SardarShahzeb** [reply]
32bcb3f955577792f737e8d2062a1954

**Reply by glitch-txs** [reply]
you need to submit the form again after testing

**Reply by SardarShahzeb** [reply]
i have done this like 5 times. test on the given link click on approval then resubmit. is there any other way to test?

**Reply by SardarShahzeb** [reply]
whenever i resubmit below rejection comes
Your update has been rejected. Reason:
"Dapp Testing - Could not verify if testing has been done on this dapp. As we do not have available V2 metric data for Sign. Please test your integration with our test wallet and submit for approval if successful. Link: https://react-web3wallet.vercel.app"
